### ISSUE TITLE
Question about the learning rate and weight decay in ResNet50 training on Imagenet

### ISSUE THREAD

**Issue by Pong-97** [alecwangcq/GraSP#5]
Hi Chaoqi,
Thanks for sharing the awesome paper and code with us. I have a little problem about the code.

in the code:
> configs/imagenet/resnet50/GraSP_80.json

"learning_rate" and "weight_decay" are both equal to 0.

could you tell me the value about this two parameters in your expriment?

Thanks,
Pong

**Comment by alecwangcq** [comment]
Hi Pong,

Sorry for the late reply.

For **ImageNet** experiments, I used [main_prune_imagenet.py](https://github.com/alecwangcq/GraSP/blob/master/main_prune_imagenet.py) for pruning and [main_finetune_imagenet.py](https://github.com/alecwangcq/GraSP/blob/master/main_finetune_imagenet.py) for training.  The config file `configs/imagenet/resnet50/GraSP_80.json` is **only** used for pruning. 

Some training details on ImageNet (see Section 5.1, Page 6 in our paper):

> Specifically, we train the pruned networks with SGD for 150 epochs, and decay the learning rate by a factor of 0.1 every 50 epochs.

> The weight decay was 1e-4. (See [main_finetune_imagenet.py#L53](https://github.com/alecwangcq/GraSP/blob/master/main_finetune_imagenet.py#L53)).

If you have any further questions, please feel free to post in this thread.

Best,
Chaoqi
